### ISSUE TITLE
Syntax error at or near "AS" for insert statement

### ISSUE THREAD

**Issue by yawhide** [go-pg/pg#210]
So I tried the following:
```
var a AnalyticsPage
err = db.Create(&a)
```
and its generating the following:
```
INSERT INTO "analytics_pages" AS "analytics_page" ("page", "referrer") VALUES ('/matchup', 'http://localhost:3000/')
```
which postgres thinks is an error in 9.4.6. I took out `AS "analytics_page"` and the stmt works in psql.

The table in postgres is called "analytics_pages"

**Comment by vmihailenco** [comment]
Looks like `AS alias` is supported only in 9.5. I will fix this, but as quick fix you can consider upgrading to 9.5.

**Comment by yawhide** [comment]
Cool thanks

**Comment by vmihailenco** [comment]
Released v4.1.5 with a fix. Thanks for the report!

**Comment by yawhide** [comment]
Np
